src/Lykke.Service.Operations.Core/Settings/AppSettings.cs

```diff
@@ -7,5 +8,6 @@ public class AppSettings
     {
         public OperationsSettings OperationsService { get; set; }
         public SlackNotificationsSettings SlackNotifications { get; set; }
+        public AssetsSettings Assets { get; set; }
```

Comment:
I think you should you existing section "AssetsServiceClient" from settings service. It already has ServiceUrl and, may be, should be extended to have CacheExpirationPeriod.

Reply:
No, better to use existing section and add it to root